src/resolvers/event.ts

```diff
@@ -195,14 +195,17 @@ export class EventResolver {
     return EventSchemas;
   }
 
-  @Authorized()
-  @Mutation(returns => Boolean)
+  @Authorized(ROLE.SCOUTMASTER, ROLE.ASST_SCOUTMASTER, ROLE.SENIOR_PATROL_LEADER, ROLE.PATROL_LEADER)
```

Comment:
We should probably allow the creator of an event to delete their own events, and not allow patrol leaders to delete others' events? Not exactly sure how the scout hierarchy works but I don't think patrol leaders should have this kind of power.